src/Plugin/ExtraField/Display/ViewUrl.php

```diff
@@ -107,8 +109,7 @@ public function getSetting($name) {
 
     if (is_callable('parent::getSetting')) {
       $value = $settings = parent::getSetting($name);
-    }
-    else {
+    } else {
```

Comment:
```suggestion
    } 
    else {
```
The previous way was as per drupal standard. Please keep it as it is.

Reply:
This has happened due to my vscode formatter, It automatically formatted this when I saved. Reverting back in next commit.